SDK doc updates

katieclay opened the pull request:
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Updating the HC docs to accommodate HC SDK updates.

katieclay committed:
First draft SDK doc updates

wsdt reviewed (changes requested):
-

katieclay committed:
Corrected installation to python

katieclay committed:
Finished corrections

katieclay committed:
Added typescript instructions

katieclay committed:
Added instructions to make server startup separate from other files

katieclay commented on .gitbook/contents/hc/2_addsub/1_write-handler.md:
````diff
@@ -6,15 +6,23 @@ The first step is to write our *off-chain handler*. This handler is responsible
+First, we'll need to import the Hybrid Compute SDK and some `Python` libraries to help with our example:
 
 ```python
+import { HybridComputeSDK } from '@bobanetwork/aa-hc-sdk-server';
````
Made some changes to address this

wsdt commented on .gitbook/contents/hc/2_addsub/1_write-handler.md:
````diff
@@ -6,15 +6,40 @@ The first step is to write our *off-chain handler*. This handler is responsible
+```typescript
+import { HybridComputeSDK } from '@bobanetwork/aa-hc-sdk-server';
+
+const sdk = new HybridComputeSDK();
````
Python + TypeScript SDK on the backend = are equal and do the same thing. They are interchangable based on the preference by the developer. 

We shouldn't mix them in one tutorial. Ideally we have something like this for "TypeScript" and "Python" when documenting the *Backend* SDK: 

![image](https://github.com/user-attachments/assets/034751f1-1e71-4199-afac-ff155afb581e)

wsdt reviewed (changes requested):
-

katieclay commented on .gitbook/contents/hc/2_addsub/1_write-handler.md:
````diff
@@ -6,15 +6,40 @@ The first step is to write our *off-chain handler*. This handler is responsible
 ```
 
-The first two, `Web3` and `eth_abi`, are established `PyPI` libraries. The third library, `offchain_utils`, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/rundler-hc/blob/boba-develop/hybrid-compute/offchain/offchain_utils.py) on our Github.
+The last two, `Web3` and `eth_abi`, are established `PyPI` libraries. The third library, `offchain_utils`, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/rundler-hc/blob/boba-develop/hybrid-compute/offchain/offchain_utils.py) on our Github.
````
Ooo good catch! Thank you!

katieclay committed:
Update .gitbook/contents/hc/2_addsub/1_write-handler.md

Co-authored-by: Riedl Kevin, Bsc. <kevin.riedl.privat@gmail.com>

katieclay committed:
Update .gitbook/contents/hc/2_addsub/1_write-handler.md

Co-authored-by: Riedl Kevin, Bsc. <kevin.riedl.privat@gmail.com>

katieclay committed:
Started python/ts tabs

katieclay committed:
Added rest of the tabs, wrote typescript section

katieclay committed:
Merge branch 'develop' into katie/HC-updates

mmontour1306 reviewed (approved):
It looks OK for the current system. Some things will change in my PR for the 0.7 version of the AA contracts, for example the version parameter passed to the off-chain handler will increase from 0.2 to 0.3 (and this requires the offchain server to use a different algorithm when generating a signature for the response). I've also renamed the "TestCounter" contract to "TestHybrid".

I'm approving this PR and I'll start a new draft for the 0.7 updates.

mmontour1306 committed:
Sync the docs with the EntryPoint v0.7 changes.
 Changes to be committed:
	modified:   .gitbook/contents/hc/2_addsub/1_write-handler.md
	modified:   .gitbook/contents/hc/2_addsub/2_write-contract.md
	modified:   .gitbook/contents/hc/2_addsub/README.md
	modified:   .gitbook/contents/hc/3_server-setup.md
	modified:   .gitbook/contents/hc/4_deploy-contract.md

katieclay committed:
Merge branch 'develop' into katie/HC-updates

katieclay committed:
Corrected client SDK reference

mmontour1306 committed:
Merge branch 'katie/HC-updates' into mm/docs-v7-update

katieclay committed:
Merge pull request #277 from bobanetwork/mm/docs-v7-update

Update docs branch for v0.7
